cloud-annotations/src/main/java/cloud/commandframework/annotations/AnnotationParser.java

```diff
@@ -406,13 +406,16 @@ public void registerCommandExecutionMethodFactory(
             final @NonNull Object instance,
             final @NonNull Collection<@NonNull CommandMethodPair> methodPairs
     ) {
+        final CommandMethod classCommandMethod = instance.getClass().getAnnotation(CommandMethod.class);
```

Comment:
Might seem random, but could you do this:
```java
final AnnotationAccessor classAnnotations = AnntationAccessor.of(instance.getClass());
final CommandMethod classCommandMethod = classAnnotation.get(CommandMethod.class);
```
? 

It'd be nice to be consistent about where we retrieve annotations from

Reply:
Fixed